dist/smartbanner.css

```diff
@@ -1,7 +1,3 @@
-/*!
- * smartbanner.js v1.5.0 <https://github.com/ain/smartbanner.js>
- * Copyright © 2017 Ain Tohvri, contributors. Licensed under GPL-3.0.
- */
```

Comment:
Please don't update distribution in PR. That will be handled directly after merge.